aana/tests/units/test_image_chat_dialog.py

```diff
@@ -0,0 +1,169 @@
+# ruff: noqa: S101
+
+from pathlib import Path
+
+import pytest
+
+from aana.core.models.image import Image
+from aana.core.models.image_chat import (
+    ImageChatDialog,
+    ImageChatMessage,
+    ImageContent,
+    TextContent,
+)
+
+IMAGE_PATH = Path("aana/tests/files/images/Starry_Night.jpeg")
+PROMPT = "describe images with more info!"
+
+def test_text_content_creation():
+    """Test that a text content can be created."""
+    content = TextContent(text=PROMPT)
+    assert content.type == "text"
+    assert content.text == PROMPT
+
+    with pytest.raises(ValueError):
+        content = TextContent(text=PROMPT, type="image")
```

Comment:
Can you add some comments why it's expected to fail?

Reply:
done